src/sauce.config.js

```diff
@@ -47,6 +47,13 @@ const overrides = {
   testIgnore: process.env.TEST_IGNORE,
 };
 
+if (process.env.BROWSER_NAME !== 'chrome') {
```

Comment:
Does chef set `chrome` or `googlechrome` as `BROWSER_NAME`?

Reply:
This is set by playwright runner https://github.com/saucelabs/sauce-playwright-runner/pull/181/files#diff-666450f3c804e34e85768192da8dae7fd947b4b14db90b48f12cea82711fcf22R346